dotnet/src/Extensions/PromptTemplates.Liquid.UnitTests/LiquidTemplateTest.ItRendersAndCanBeParsedAsync.verified.txt

```diff
@@ -0,0 +1,3 @@
+﻿system:This is the system message
```

Comment:
Where are the *.verified.txt files used? Are unit tests consuming them, or perhaps generating them? If it's the former, why are they not part of the project? If it's the latter, why is this necessary?

Reply:
It's consumed by `Verify`, see https://github.com/microsoft/semantic-kernel/pull/6048#discussion_r1585139820